src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
```

Comment:
create a private final variable of the Random class instead of creating a new one.
Also, the maximum ball number would be 99. See Random class [documentation ](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html)for more information.